dali/operators/audio/mfcc/mfcc.cu

```diff
@@ -69,7 +69,7 @@ bool MFCC<GPUBackend>::SetupImpl(std::vector<OutputDesc> &output_desc,
     output_desc = detail::SetupKernel<T>(kmgr_, ctx_, input, make_cspan(args_), axis_);
   ), DALI_FAIL(make_string("Unsupported data type: ", input.type().id())));  // NOLINT
   int64_t max_ndct = 0;
-  for (int i = 0; i < nsamples_; ++i) {
+  for (int i = 0; i < output_desc[0].shape.num_samples(); ++i) {
```

Comment:
why this change?

Reply:
I removed the member field, the coverity was complaining that it is not initialized, and I didn't see it in @szalpal PR, so decided to go with the flow of the rework.